components/atoms/CardGrid/CardGrid.tsx

```diff
@@ -4,10 +4,16 @@ import styles from "./CardGrid.module.css";
 
 type CardGridProps = {
   children: React.ReactNode;
+  className?: String;
 };
 
-const CardGrid = ({ children }: CardGridProps) => {
-  return <div className={styles.grid}>{children}</div>;
+const CardGrid = ({ children, className }: CardGridProps) => {
+  const customClass = className;
+  return (
+    <div className={className ? styles[`${customClass}`] : styles.grid}>
```

Comment:
This might be a good instance to make use of the classnames library: https://www.npmjs.com/package/classnames